src/arch/riscv/isa.cc

```diff
@@ -331,6 +334,60 @@ ISA::setMiscReg(int misc_reg, RegVal val)
         warn("Ignoring write to %s.\n", CSRData.at(misc_reg).name);
     } else {
         switch (misc_reg) {
+
+          case MISCREG_PMPCFG0:
```

Comment:
Can you explain this fallthrough. It's probably obvious from the spec, but why do we handle CFG0 and CFG2 the same (what about CFG1??). A brief statement and a pointer to the spec is probably enough.

Reply:
I have added a comment in the code. According to specs, the odd-numbered configuration registers are illegal for RV64 and each 64 bit CFG register hold configurations for 8 PMP entries. The only difference in handling of these regs, will be passing the correct pmp index to pmpUpdateCfg function(), which I am trying to do through this expression "i+(4*(misc_reg-MISCREG_PMPCFG0)" where i refers to a PMP entry number.